google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -728,7 +728,7 @@ public ReadChannel reader(BlobSourceOption... options) {
    * try (WriteChannel writer = blob.writer()) {
```

Comment:
no need for the nested try block, one will do here

Reply:
Done